### PR TITLE
Yurtctl devfor ut

### PR DESCRIPTION
**Pull request by River-sh**
/kind enhancement

#### Which issue(s) this PR fixes:
 
Fixes #

#### Does this PR introduce a user-facing change?
 Yurttest is the main function in yurtctl, and the unit test coverage of yurttest has been  promoted to 56.1%.
```release-note

```

### CODE REVIEW

**Comment by rambohe-ch** [comment]
@River-sh please fix the pipeline errors.

File: pkg/controller/nodelifecycle/node_lifecycle_controller.go [hunk 1]
```diff
@@ -1435,6 +1437,7 @@ func (nc *Controller) cancelPodEviction(node *v1.Node) bool {
 //   Returns false if the node name was already enqueued.
 // - deletes pods immediately if node is already marked as evicted.
 //   Returns false, because the node wasn't added to the queue.
+
 func (nc *Controller) evictPods(node *v1.Node, pods []*v1.Pod) (bool, error) {
 	// if node is in autonomy status, skip evict pods from the node
 	if node != nil && node.Annotations != nil && node.Annotations[nodeutil.AnnotationKeyNodeAutonomy] == "true" {
```
Comment (Congrool) [inline pkg/controller/nodelifecycle/node_lifecycle_controller.go:1443]: Do we need to modify this file?

File: pkg/controller/nodelifecycle/node_lifecycle_controller_test.go [hunk 2]
```diff
@@ -3112,24 +3112,25 @@ func TestApplyNoExecuteTaintsToNodesEnqueueTwice(t *testing.T) {
...
+		}
+			node5, err := fakeNodeHandler.Get(context.TODO(), "node5", metav1.GetOptions{})
+			if err != nil {
+				t.Errorf("Can't get current node5...")
+				return
+			}
+			if !taintutils.TaintExists(node5.Spec.Taints, NotReadyTaintTemplate) || len(node5.Spec.Taints) == 0 {
+				t.Errorf("Not found taint %v in %v, which should be present in %s", NotReadyTaintTemplate, node5.Spec.Taints, node5.Name)
+			}
+	*/
```
[the remark this answers is not in the document]
Comment (River-sh) [reply pkg/controller/nodelifecycle/node_lifecycle_controller_test.go]: The test case does not pass using novel version go, which does not pass at all in GO1.18 and sometimes in GO1.16.15 in reality.

**Review by rambohe-ch — APPROVED** [review]

File: pkg/controller/nodelifecycle/node_lifecycle_controller.go (hunk 1 above)
Comment (River-sh) [reply pkg/controller/nodelifecycle/node_lifecycle_controller.go]: Inconsistent versions of golint resulted in mistaken changes, which have been restored.

**Comment by rambohe-ch** [comment]
Fixes #920

**Pull request merged**